.terraform/jenkins-agents-images/main.tf

```diff
@@ -0,0 +1,43 @@
+variable "terraform-images" {
+  type = map
+  default = {
+    "terraform-bionic" = {
+      "os_type"  = "Linux"
+      "blob_uri" = "https://oejenkinswesteurope.blob.core.windows.net/disks/jenkins-terraform-ubuntu-18.04.vhd"
+    }
+    "terraform-xenial" = {
+      "os_type"  = "Linux"
+      "blob_uri" = "https://oejenkinswesteurope.blob.core.windows.net/disks/jenkins-terraform-ubuntu-16.04.vhd"
+    }
+    "terraform-ubuntu-nonSGX" = {
+      "os_type"  = "Linux"
+      "blob_uri" = "https://oejenkinswesteurope.blob.core.windows.net/disks/jenkins-terraform-ubuntu-nonSGX.vhd"
+    }
+    "terraform-win2016" = {
+      "os_type"  = "Windows"
+      "blob_uri" = "https://oejenkinswesteurope.blob.core.windows.net/disks/jenkins-terraform-win-2016.vhd"
+    }
+    "terraform-win2016-dcap" = {
+      "os_type"  = "Windows"
+      "blob_uri" = "https://oejenkinswesteurope.blob.core.windows.net/disks/jenkins-terraform-win-dcap.vhd"
```

Comment:
These URLS should be specific to 2016, as 2019 is on the horizon.
`
jenkins-terraform-win-2016-dcap.vhd`